code/modules/mob/living/carbon/human/human.dm

```diff
@@ -284,12 +287,12 @@
 			else
 
 				var/limbs_affected = pick(0, 1)
-				var/datum/organ/external/processing_dismember
+				var/obj/item/organ/external/processing_dismember
 
 				while(limbs_affected != 0)
 					processing_dismember = pick(organs)
 					if(processing_dismember.name != "chest" && processing_dismember.name != "head")
-						processing_dismember.droplimb(1,0,1)
+						processing_dismember.droplimb(0,1)

```

Comment:
These not being amputations are going to make moderate explosions deal absurd damage levels of ~160+ while giving an incredibly high chance to break an additional bone (~75%). Damage from *light* explosions will typical clock in at ~70 with a high probability (50%) of breaking an additional bone.

The first will put someone into out and out full crit or deep depp softcrit with the oh-so-fun permastun of it not too far out. 

The second will put people into soft-crit 50% of the time with something that's otherwise meant to be a mild inconvenience.